isdown/manifest.json

```
@@ -0,0 +1,47 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "98d8eda3-7bc5-4067-8ff6-63d11d0132e2",
+  "app_id": "isdown",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "IsDown helps companies monitor all third-party status pages in one place",
+    "title": "IsDown",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "IsDown and Datadog Flow",
+        "image_url": "images/isdown_datadog_flow.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "IsDown Dashboards",
+        "image_url": "images/isdown_dashboards.jpg"
+      }
+    ],
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS",
+      "Offering::Integration",
+      "Category::Notification",
+      "Category::Monitoring",
+      "Category::Cloud"
+    ]
+  },
+  "assets": {
```

@alopezz it looks like integration assets are missing (events, configuration, and source_type_name). Could you provide guidance on how to resolve this?

---

Right, the relevant docs are [here](https://docs.datadoghq.com/developers/integrations/check_references/?tab=manifestversion2#manifest-file).

This is probably what is needed for this integration:

```suggestion
  "assets": {
    "integration": {
      "source_type_name": "isdown",
      "configuration": {},
      "events": {
        "creates_events": true
      },
      "metrics": {
        "prefix": "isdown.",
        "check": [],
        "metadata_path": "metadata.csv"
      },
      "service_checks": {
        "metadata_path": "assets/service_checks.json"
      }
    },
```

---

@ntomas Could you commit the suggestion above? Thanks!

---

@emarsha94 done!